i3-agenda.py

```diff
@@ -16,6 +16,8 @@
 TMP_TOKEN = '/tmp/i3agenda_google_token.pickle'
 CACHE_PATH = '/tmp/i3agenda_cache.txt'
 
+allowed_calendars_ids = [] # populate with the Google Calendar IDs you are interested in
```

Comment:
Would be even better if the IDs could be passed as command line arguments, this way users could use their IDs and update the repo easily if changes will occur